source/camera/basic/capture_assistant.py

```diff
@@ -1,21 +1,39 @@
-"""Capture Assistant sample."""
+"""
+This example shows how to use Capture Assistant to capture point clouds, with color, from the Zivid camera.
+"""
+
 import datetime
 import zivid
 
 
 def _main():
     app = zivid.Application()
+
+    print("Connecting to camera")
     camera = app.connect_camera()
 
     suggest_settings_parameters = zivid.capture_assistant.SuggestSettingsParameters(
         max_capture_time=datetime.timedelta(milliseconds=1200),
         ambient_light_frequency=zivid.capture_assistant.SuggestSettingsParameters.AmbientLightFrequency.none,
     )
 
+    print(f"Running Capture Assistant with parameters: {suggest_settings_parameters}")
     settings = zivid.capture_assistant.suggest_settings(camera, suggest_settings_parameters)
 
-    with camera.capture(settings) as hdr_frame:
-        hdr_frame.save("Result.zdf")
+    print("Settings suggested by Capture Assistant:")
+    for acquisition in settings.acquisitions:
+        print(acquisition)
+
+    print("Manually configuring processing settings (Capture Assistant only suggests acquisition settings)")
```

Comment:
Not true. We also suggest filters (it's just that the suggested filters are always the same).

I do like that you demonstrate that it is possible to adjust settings even after running the CA though, so don't remove that :)

Reply:
Sould I then right something like (filter settings returned by the current version of Capture Assistant are always the same)?
Or should I just keep it as it is? I boils down to the same thing. Except that the CA will overwrite filter settings if they are set before right? That is the issue?